Allos.Amazon.Sdk/S3/Transfer/Commands/MultipartUploadCommand.cs

```diff
@@ -101,7 +96,7 @@ public override async Task ExecuteAsync(CancellationToken cancellationToken)
                 ArgumentNullException.ThrowIfNull(_fileTransporterRequest.ContentLength);
                 
                 var initRequest = ConstructInitiateMultipartUploadRequest();
-                var initResponse = await _s3Client.InitiateMultipartUploadAsync(initRequest, cancellationToken)
+                var initResponse = await S3Client.InitiateMultipartUploadAsync(initRequest, cancellationToken)
```

Comment:
there is a foreach below that uses `localThrottler`. Need to release is in the loop

```
foreach (var uploadRequest in _partsToUpload)
{
	await localThrottler.WaitAsync(cancellationToken)
		.ConfigureAwait(continueOnCapturedContext: false);

	try
	{
		// Perform upload
	}
	finally
	{
		localThrottler.Release();
	}
}
```

Reply:
Nice find!  =D =D